Convert.pm

```diff
@@ -103,17 +126,18 @@ sub date2time {
 # Date format:  1999:260:03:30:01.542
 # 956245305.5 = eng_decom (unix) time at VCDU = 4324480 around 2000:111:15:41:46 (2000-04-20T15:42:50)
 # CXC Time from CCDM file is 72632569.96
-
     my $date = shift;
-    my $t1998 = @_ ? 0.0 : 956245305.5- 72632569.96 + 1.276; # 2nd argument implies Unix time not CXC time
+    # second argument can be used to request return time as unix seconds
+    my $want_unix_secs = shift;
+    # if this is delta date from the DOT, just return it in seconds
     my ($sec, $min, $hr, $doy, $yr) = reverse split ":", $date;
-
-    return ($doy*86400 + $hr*3600 + $min*60 + $sec) unless ($yr);
-
-    my ($mon, $day) = ydoy2md($yr, $doy);
-
-    $sec = 59 if ($sec > 59);
-    return timegm($sec,$min,$hr,$day,$mon-1,$yr) - $t1998;
+    if (not $yr){
```

Comment:
I think this is now equivalent to the previous 'unless ($yr)' though I don't know if we would be better off counting the pieces that you get if you split on the colons.